### PR TITLE
Earl grey demo

### PR DESCRIPTION
**Pull request by amlcurran**
## This PR adds

A demo of Earl Grey and a wrapper API around it

## Considerations and implementation

The Earl Grey API is a bit minging; this changes the old one we used in #1 which is based on Swift 2.3 and an older version of Earl Grey. This will be featured on a blog post next week about UI testing

### Test(s) added 

The entire thing is about tests!

### Screenshots

See the blog post next week

### Paired with 

@eduardourso and @FabiaS when we first wrote it

### CODE REVIEW

**Comment by takecare** [comment]
no readme? :( [context](http://google.github.io/EarlGrey/) for those who weren't aware of earl grey (like me)

**Comment by amlcurran** [comment]
@takecare 😱 added details to the main README, thanks for catching!

File: Earl-Grey-Demo/Earl-Grey-Demo/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        window = UIWindow()
+        window?.rootViewController = UINavigationController(rootViewController: ViewController())
+        window?.makeKeyAndVisible()
+        return true
+    }
+
+    func applicationWillResignActive(_ application: UIApplication) {
```
Comment (sergdort) [inline Earl-Grey-Demo/Earl-Grey-Demo/AppDelegate.swift]: I think we can remove these empty methods 😜

File: Earl-Grey-Demo/Earl-Grey-Demo/SecondViewController.swift [hunk 2]
```diff
@@ -0,0 +1,40 @@
+//
+// Created by Alex on 21/02/2017.
+// Copyright (c) 2017 Alex Curran. All rights reserved.
+//
```
Comment (dominicfreeston) [inline Earl-Grey-Demo/Earl-Grey-Demo/SecondViewController.swift]: Should we remove comments?

File: Earl-Grey-Demo/Earl-Grey-Demo/AppDelegate.swift (hunk 1 above)
Comment (amlcurran) [inline Earl-Grey-Demo/Earl-Grey-Demo/AppDelegate.swift]: yes! https://github.com/novoda/ios-demos/pull/4/commits/50a215f9db2272e323592576cdd5a3383347067e

File: Earl-Grey-Demo/Earl-Grey-Demo/SecondViewController.swift (hunk 2 above)
Comment (amlcurran) [inline Earl-Grey-Demo/Earl-Grey-Demo/SecondViewController.swift]: yup, also added a licence to the wrapper API so others can use it https://github.com/novoda/ios-demos/pull/4/commits/fe3327e86467ad661b298e3b7694a2659cd31cf6

**Comment by amlcurran** [comment]
@dominicfreeston updated to remove pods, and add instructions to the README to reflect this https://github.com/novoda/ios-demos/pull/4/commits/6cd96f18b1fbbcbf35af5581c291a11f4ccfdabb

**Pull request merged**